src/Storage/Database/Prefill/RecordContentGenerator.php

```diff
@@ -278,12 +278,15 @@ private function addText(Entity\Content $contentEntity, $fieldName, $type, array
      */
     private function addJson(Entity\Content $contentEntity, $fieldName, $type)
     {
+        $contentType = $this->repository->getEntityManager()->getContentType($contentEntity->getContenttype());
+        $placeholder = isset($contentType['fields'][$fieldName]['placeholder']) ? $contentType['fields'][$fieldName]['placeholder'] : null;
+
         $value = null;
```

Comment:
While I am nagging, drop the newline before this one :stuck_out_tongue_winking_eye: